pf/README.md

```diff
@@ -200,20 +192,20 @@ to the Plugin Framework.
      }
      ```
 
-4. Update code declaring `tfbridge.ProviderInfo` (typically in `provider/resources.go`) from
-   `github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge` and to declare `ProviderInfo` from
-   `github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge` instead.
+4. Update code declaring `tfbridge.ProviderInfo` (typically in `provider/resources.go`):
 
     ```go
 
     //go:embed cmd/pulumi-resource-myprovider/bridge-metadata.json
     var bridgeMetadata []byte
 
-    func Provider() pf.ProviderInfo {
+    func Provider() tfbridge.ProviderInfo {
         info := tfbridge.ProviderInfo{
-            // Comment out P, use NewProvider below instead.
+            // Comment out P
```

Comment:
Why is this "Comment out P"? Couldn't we just say "Replace P: shimv2.NewProvider(...) with pf.ShimProvider(shim.NewProvider())"?

Reply:
Good catch.